### DISCUSSION TITLE
I run the resulting .app, but it quits immediately.

### DISCUSSION THREAD

**Discussion by developers-dev** [dashingsoft/pyarmor#1575] [Q&A]
<img width="326" alt="image" src="https://github.com/dashingsoft/pyarmor/assets/55382583/4fd1eaa9-7ee9-4b27-8fd2-ae6b7764a60b">
Pyside6 Project

what's the problem?

Pyamror 7.7.4 (pyarmor-vax-)
Pyinstaller 5.7.0
Pyinstaller-hooks-contrib 2023.10
Python 3.8.16 :: Intel Corporation

pyarmor pack --clean -e "--noconsole --icon=data.icns --onefile " main.py

**Comment by jondy** [comment]
Refer to
https://pyarmor.readthedocs.io/en/v7.7/questions.html#the-final-bundle-does-not-work
